light-client/src/supervisor.rs

```diff
@@ -10,32 +15,21 @@ use crate::{
     types::{Height, LightBlock, PeerId, Status},
 };
 
-use tendermint::evidence::{ConflictingHeadersEvidence, Evidence};
-
-use contracts::pre;
-use crossbeam_channel as channel;
-
 /// Type alias for readability
 pub type VerificationResult = Result<LightBlock, Error>;
```

Comment:
Since we are using a fully expanded type in `HandleInput::LatestTrusted`, I think we can remove this type alias.